backend/vaa-strapi/src/api/candidate/content-types/candidate/lifecycles.ts

```diff
@@ -0,0 +1,7 @@
+import crypto from 'crypto';
+
+export default {
+  beforeCreate(event) {
+    event.params.data.registrationKey = crypto.randomUUID();
```

Comment:
Maybe worth testing, but this could also overwrite the explicitly configured registration key when importing (perhaps this should be `event.params.data.registrationKey ?? crypto.randomUUID()` instead if we're not able to overwrite this?)